packages/server/graphql/mutations/setTaskEstimate.ts

```diff
@@ -209,17 +212,32 @@ const setTaskEstimate = {
         return {error: {message: 'User no longer has access to Azure DevOps'}}
       }
 
+      let fieldName = SprintPokerDefaults.SERVICE_FIELD_COMMENT.toString()
+      let fieldType = 'string'
       if (!azureDevOpsDimensionFieldMapEntry) {
-        return {error: {message: 'Cannot find the correct field to push changes to.'}}
+        try {
+          const props = {
+            teamId,
+            dimensionName,
+            fieldName: fieldName,
+            fieldId: fieldName,
+            instanceId,
+            fieldType: fieldType,
```

Comment:
+1 we could just do `fieldType,` here and `fieldName,` like the other values with the same name as the keys